game_board.py

```diff
@@ -193,9 +208,106 @@ def place_stone(plus_list,current_row,stone_marker):
     plus_list -- the lists of all the "+" and their positions
     stone_marker -- the char of the stone, should be either "X" or "O" 
     """
-    plus_list[current_row][0] = stone_marker
+    current_char = plus_list[current_row][0]
+    if current_char == "+":
+        plus_list[current_row][0] = stone_marker
     return plus_list
 
+def remove_stone(map_coordinates,stone_marker):
+    """ When you place a stone it removes the stone from the left or the right side
+        depending on the stone_marker
+    Returns map_coordinates 
+    Keyword arguments:
+    map_coordinates -- list of all chars in the string version of the map
+    stone_marker -- Char version of the stone which can be a "X" or "O"     
+
+    """
+    for stone in map_coordinates:
+        if stone[0] == stone_marker:
+            stone[0] = ""
+            break
+    return  map_coordinates
+def which_stone(player_turn):
+    """Picks which char the stone marker should be based on which player is next to act
+    Return stone_marker
+
+    Keyword arguments:
+    stone_marker -- char of the stone,which can be "X" or "O"
+    """
+    stone_marker = "X"
+    if player_turn == False:
+        stone_marker = "O"
+    return stone_marker    
+
+def switch_player_turn(player_turn):
+    """Switches which player turn it is to act
+    Return player_turn
```

Comment:
You should explain what the return value will contain i.e. explaining that it will switch to the other players turn

Reply:
Fixed